2d/camera-zoom/Actors/ZoomingCamera2D.gd

```diff
@@ -9,6 +11,13 @@ export var max_zoom := 2.0
 var _zoom_level := 1.0 setget _set_zoom_level
 
 onready var tween: Tween = $Tween
+onready var zoom_slider: VSlider = get_node(
+	"/root/ZoomingCameraDemo/UI/Zoom_slider"
```

Comment:
That's a code smell. We shouldn't teach directly coupling gameplay logic to UI (same as with any programming: don't couple business logic with UI, otherwise your app's core will break when editing the UI). The guideline is to only get path to child nodes. Instead, use a parent to wire the two nodes.